io.catenax.battery.battery_pass/3.0.1/BatteryPass.ttl

```diff
@@ -0,0 +1,920 @@
+    bamm:properties (:cycleLifeTestCRate :cycleLifeTestDepthOfDischarge :expectedLifetime);
+    bamm:preferredName "battery cycle life entity"@en;
+    bamm:description "Entity to bundle the characterisitics describing the cycle life of a battery"@en.
+:packagingInstructions a bamm:Property;
+    bamm:name "packagingInstructions";
+    bamm:preferredName "packaging instructions"@en;
+    bamm:description "Instructions for safely packaging\nbatteries is describing a business requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:transportationInstructions a bamm:Property;
+    bamm:name "transportationInstructions";
+    bamm:preferredName "transportation instructions"@en;
+    bamm:description "Instructions for safely transporting\nbatteries is describing a business requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:vehicleDismantlingProcedure a bamm:Property;
+    bamm:name "vehicleDismantlingProcedure";
+    bamm:preferredName "vehicle dismantling procedure"@en;
+    bamm:description "Document containing the vehicle dismantling procedure is describing a regulatory requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:batteryDismantlingProcedure a bamm:Property;
+    bamm:name "batteryDismantlingProcedure";
+    bamm:preferredName "battery dismantling procedure"@en;
+    bamm:description "Document containing the battery dismantling procedure is describing a regulatory requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:safetyMeasures a bamm:Property;
+    bamm:name "safetyMeasures";
+    bamm:preferredName "safety measures"@en;
+    bamm:description "Safety measures document(s) is describing a regulatory requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:testReportsResults a bamm:Property;
+    bamm:name "testReportsResults";
+    bamm:preferredName "test reports results"@en;
+    bamm:description "Results of test reports which prove that the battery fulfills this regulation and its delegated regulations is describing a regulatory requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:declarationOfConformity a bamm:Property;
+    bamm:name "declarationOfConformity";
+    bamm:preferredName "declaration of conformity"@en;
+    bamm:description "Declaration of conformity (CE) is describing a regulatory requirement."@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:responsibleSourcing a bamm:Property;
+    bamm:name "responsibleSourcing";
+    bamm:preferredName "responsible sourcing"@en;
+    bamm:description "Document/Certificates on organizations compliance to ethical business practices"@en;
+    bamm:characteristic :AttachmentLinkCharacteristic.
+:name a bamm:Property;
+    bamm:name "name";
+    bamm:preferredName "name"@en;
+    bamm:description "Name of the manufacturer is describing a regulatory requirement."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "Samsung".
```

Comment:
would not use trademark as example value. Use maybe ACME corp.

Reply:
resolved